bokehjs/src/coffee/renderer/glyph/text.coffee

```diff
@@ -21,15 +25,10 @@ class TextView extends Glyph.View
 
   draw_legend: (ctx, x1, x2, y1, y2) ->
     ctx.save()
-    reference_point = @get_reference_point()
-    if reference_point?
-      glyph_settings = reference_point
-    else
-      glyph_settings = @props
-    text_props = @visuals.text
-    text_props.set(ctx, glyph_settings)
+    @text_props.set_value(ctx)
     # override some features so we fit inside the legend
-    ctx.font = text_props.font(12)
+    ctx.font = @text_props.font_value()
+    ctx.font = ctx.font.replace(/\b[\d\.\d]+[\w]+\b/, '10pt')
```

Comment:
It's meant to cover the case that somebody passes the ".5pt" as a text_font_size. Otherwise, it can't hurt I think?